akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -118,18 +123,19 @@ private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_])
   def within[T](max: JDuration)(f: Supplier[T]): T =
     within_internal(Duration.Zero, max.asScala.dilated, f.get())
 
-  private def within_internal[T](min: FiniteDuration, max: FiniteDuration, f: ⇒ T): T = {
+  private def within_internal[T](min: FiniteDuration, max: FiniteDuration, f: => T): T = {
     val start = now
     val rem = if (end == Duration.Undefined) Duration.Inf else end - start
     assert(rem >= min, s"required min time $min not possible, only ${rem.pretty} left")
 
     lastWasNoMessage = false
 
-    val maxDiff = max min rem
+    val maxDiff = max.min(rem)
```

Comment:
I guess we just accept this style.

Reply:
I think @patriknw pushes for that over spaces

Reply:
I prefer `abc.map {` over `abc map {`
For some short operator style method names like `min` it can also be good, but one have to choose.

If we dislike this, we could run one sweep to get rid of infix and then turn off that formatting rule and manually add back infix where it makes sense.

Reply:
maybe `rewrite.neverInfix.excludeFilters` could work

Reply:
`rewrite.neverInfix.excludeFilters` worked great